HHAuto.user.js

```diff
@@ -3822,6 +3822,7 @@ function goAndCollectDailyGoals()
             setStoredValue("HHAuto_Temp_autoLoop", "false");
             let buttonsToCollect = [];
             const listDailyGoalsTiersToClaim = $("#daily_goals .progress-section .progress-bar-rewards-container .progress-bar-reward");
+            let potionsNum = Number($('.progress-section div.potions-total > div > p').text());
```

Comment:
What is this potionsNum ?

Reply:
@deuxge it's the current number of achieved daily goals
![Capture](https://user-images.githubusercontent.com/7374316/231671175-51a47e4a-5307-42f0-aa21-a79d9ddc88f2.JPG)